### PR TITLE
Update to latest CocoaPods (1.9.1) and fix incremental installation integration.

### PR DESCRIPTION
**Pull request by dnkoutso**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -7,9 +7,10 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
...
 
 gem 'clintegracon', '~> 0.9.0'
-gem 'cocoapods', '~> 1.8.beta', github: 'CocoaPods/CocoaPods'
+gem 'cocoapods', '~> 1.9', github: 'CocoaPods/CocoaPods'
 gem 'cocoapods-core', github: 'CocoaPods/Core'
 gem 'inch_by_inch', '~> 1.0.1'
 gem 'prettybacon'
 gem 'rspec', '~> 3.9'
 gem 'rubocop', '~> 0.55.0'
+gem 'activesupport', '= 4.2.11'
```
Comment (dnkoutso) [inline Gemfile:16]: fixes an issue in Travis CI that 4.2.11.14 of activesupport not found.

File: lib/cocoapods/generate/installer.rb [hunk 2]
```diff
@@ -92,19 +97,24 @@ def open_app_project(recreate: false)
       # @return [Xcodeproj::Project]
       #
       def create_app_project
-        app_project = open_app_project(recreate: true)
+        app_project = open_app_project(recreate: !configuration.incremental_installation?)
+        # Do not re-integrate the "user" project if it already exists and we are performing an incremental installation.
+        return if app_project.path.exist? && configuration.incremental_installation?
```
Comment (amorde) [inline lib/cocoapods/generate/installer.rb:102]: Couldn't there still be an incremental change that requires updating the user project?

File: Gemfile [hunk 3]
```diff
@@ -6,8 +6,9 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}" }
 
 gemspec
 
+gem 'activesupport', '= 4.2.11'
```
Comment (dnkoutso) [inline Gemfile:9]: see https://github.com/square/cocoapods-generate/pull/49#discussion_r403371631

File: lib/cocoapods/generate/configuration.rb [hunk 4]
```diff
@@ -316,6 +316,19 @@ def use_frameworks?
...
+      # @return [String] The project name to use for generating this workspace.
+      #
+      # @param [Specification] spec
+      #        the specification to generate project name for.
+      #
+      def project_name_for_spec(spec)
+        project_name = spec.name.dup
+        # When using multiple Xcode project the project name will collide with the actual .xcodeproj meant for the pod
+        # that we are generating the workspace for.
+        project_name << 'Sample' if generate_multiple_pod_projects?
```
Comment (amorde) [inline lib/cocoapods/generate/configuration.rb:328]: can we make this somehow related to `spec.name` so it's visually related? `"#{spec.name}-Gen"` or something similar

File: lib/cocoapods/generate/installer.rb (hunk 2 above)
Comment (dnkoutso) [reply lib/cocoapods/generate/installer.rb:102]: talked offline. CocoaPods will handle that internally for the user projects as well. However, I can remove this check and instead avoid writing the native targets if they already exist.

File: lib/cocoapods/generate/configuration.rb (hunk 4 above)
Comment (dnkoutso) [reply lib/cocoapods/generate/configuration.rb:328]: it already includes `spec.name` we just append 'Sample' for multi-xcodeproj.

File: lib/cocoapods/generate/installer.rb (hunk 2 above)
Comment (dnkoutso) [reply lib/cocoapods/generate/installer.rb:102]: updated and now we always write the user project and avoid re-creating its native targets.

File: lib/cocoapods/generate/configuration.rb (hunk 4 above)
Comment (dnkoutso) [reply lib/cocoapods/generate/configuration.rb:328]: do you mean just the suffix to append to be different?

Comment (amorde) [reply lib/cocoapods/generate/configuration.rb:328]: 🤦‍♂ totally misread this, thanks. missed the `<<`

**Pull request merged**